2Dgame/Assets/Scripts/Platform Scripts/BlinkerScript.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BlinkerScript : PlatformScript {
+
+    new Renderer renderer;
+
+    private new void Awake()
+    {
+        renderer = GetComponent<Renderer>();
+    }
+
+    /**
+     * Enables the renderer when jumped on
```

Comment:
Check to see if the blinkerprefab has the correct script attached, I thought i fixed it in the add correct script commit. I may have unfixed it when i reset the basicplatformscheme....